oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -77,19 +77,11 @@ Archive Steps
 
 These steps should be followed for all repos within the edX organization(forks included). After some experiments with keeping archived repos in the ``edx`` organization, we've learned that having abandoned code show up in searches hinders work to understand the current state of the system and the risk around new work, particularly deprecations and API changes. Thus we decided to move all archived repositories to a separate org.
 
-1. Update the README.rst file in the repository to state that it is archived, using the `README Archive Statement`_ below.
+1. (Recommended) Update the README.rst file in the repository to state that it is archived, using the `README Archive Statement`_ below.
 
-2. Update the openedx.yaml file, creating it if necessary:
+2. Unless you have the relevant permissions to the work in this step, create an IT help ticket and ask them to do the following:
 
-   - Add ``archived: True``.
-
-   - Remove the ``openedx-release`` key if it is present.
```

Comment:
**Author Concern:** Is it still important to remove the `openedx-release` key if this is going to the unsupported org?

Reply:
No, I think we can remove both of these openedx.yaml changes. I've included a commit to update OEP 2 as well.